app/bundles/FormBundle/Entity/Submission.php

```diff
@@ -86,7 +86,7 @@ public static function loadMetadata(ORM\ClassMetadata $metadata)
 
         $builder->addIpAddress();
 
-        $builder->addLead(true, 'SET NULL');
+        $builder->addLead(false, 'SET NULL');
```

Comment:
This changes the table schema. Could you please give some details on the need for this change?